src/main/java/wolox/training/client/models/AuthorDto.java

```diff
@@ -0,0 +1,15 @@
+package wolox.training.client.models;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class AuthorDto {
```

Comment:
It is ok for this use case, but if you had an Author in your database model before having this DTO for the author external API model, this class probably will conflict with that one.